quilt/src/main/resources/quilt.mod.json

```diff
@@ -16,10 +16,6 @@
         "id": "quilt_loader",
         "version": "*"
       },
-      {
-        "id": "quilt_base",
-        "version": "*"
-      },
```

Comment:
why is this being removed?

Reply:
We don't need this as a dependency now as we include it in the mod